tests/cypress/integration/taxonomy.test.js

```diff
@@ -49,18 +50,26 @@ describe('Admin can create and update podcast taxonomy', () => {
 		cy.get('.row-title').should('have.text', 'Distributed');
 	});
 
-	it('Can delete taxonomy', () => {
-		cy.visit(
-			'/wp-admin/edit-tags.php?taxonomy=podcasting_podcasts&podcasts=true'
-		);
-		cy.get('.row-title').should('have.text', 'Distributed').click();
-		cy.url().should('contains', 'http://localhost:8889/wp-admin/term.php');
-		cy.on('window:confirm', () => true);
-		cy.get('.delete').click();
-		cy.url().should(
-			'contains',
-			'http://localhost:8889/wp-admin/edit-tags.php'
-		);
-		cy.get('.wp-list-table').should('contain.text', 'No podcasts found');
-	});
+	if (Cypress.env('HAS_BLOCKS_EDITOR')) {
```

Comment:
@dinhtungdu why are we conditionally testing this "detele term" workflow?

Reply:
WP 4.6 doesn't have the delete link on the edit term detail page. That's the reason why for the exclusion. But I agree that the environment variable name makes it confusing here.